INSTALL

```diff
@@ -65,3 +65,6 @@ The html version can be build with the command:
 
 For generating the documentation it is assumed that a Tex system is installed.
 
+The testsuite can be run with:
+
+   (asdf:test-system :cl-cffi-gtk)
```

Comment:
Does this actually work for you? Because nothing happens with that call, it only works for me when I use the `:cl-cffi-gtk-test` as argument.

Reply:
Yes it works for me when calling `asdf:test-system` on the main system (and not only on the `-test` one). There is some code in `cl-cffi-gtk.asd` that ensures the link between the two systems (you can see for example that `alexandria.asd` uses the same trick).

I don't know why it does not work for you. Maybe your ASDF is too old? I've tested this on 3.1.7.